integration-tests/testsetups/ccip/test_helpers.go

```diff
@@ -224,7 +225,18 @@ func NewLocalDevEnvironment(
 	for _, chain := range allChains {
 		timelocksPerChain[chain] = state.Chains[chain].Timelock
 		tokenInfo := tokenConfig.GetTokenInfo(env.Logger, state.Chains[chain].LinkToken, state.Chains[chain].Weth9)
-		ocrParams[chain] = changeset.DefaultOCRParams(feedSel, tokenInfo)
+
+		params := changeset.DefaultOCRParams(feedSel, tokenInfo)
+
+		if tCfg.IsFeeBoosting {
+			// Override specific parameters for fee boosting scenarios
+			params.ExecuteOffChainConfig.RelativeBoostPerWaitHour = 10
+			params.CommitOffChainConfig.TokenPriceBatchWriteFrequency = *config.MustNewDuration(1_000_000 * time.Hour)
+			params.CommitOffChainConfig.RemoteGasPriceBatchWriteFrequency = *config.MustNewDuration(1_000_000 * time.Hour)
+			params.CommitOffChainConfig.TokenInfo = nil
+		}
```

Comment:
Can we make this more general? Like be able to override anything in the OCR params via the `TestConfigs` object?

Reply:
Simple idea:

```golang
type TestConfigs struct {
// .. previous fields
  OCRConfigOverride func(changeset.CCIPOCRParams) changeset.CCIPOCRParams
}

// in the above test_helpers.go file
params := changeset.DefaultOCRParams(feedSel, tokenInfo)
if tCfg.OCRConfigOverride != nil {
  params = tCfg.OCRConfigOverride(params)
}
```

Reply:
I like the idea 💡